AutoStep/pPump.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+* This program is contains functions for the pPump library
+* Programmers: CJD
+* Editor: 
+* Iteration: 0.2
+* Last edited: 02/03/2021
+* Created: 10/08/2021
+* 
+* Dependencies: pPump.h, Arduino.h
+*
+* Library contrains fuctions for driving peristaltic pumps as they will be used for Boozebot
+* (Will be expanded to include linear rail functions and protothreading in next iteration)
+*/
+
+#include "Arduino.h"
+#include "pPump.h"
+
+pPump::pPump(int stepPin, int dirPin, int buttonPin, int period)
```

Comment:
Constructor looks and correctly implemented. Constructor clear initializes pins and speed(period) for each pump